### ISSUE TITLE
Log error editing related content after migrate the relationship and push 

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#16217]
When you have a migrated relationship, and you push it to the receiver side, every time you try to open or edit the piece of content, we are getting an error in the logs 

## Expected Behavior
We need be able to continue navigating without errors 

## Current Behavior
Every time you open//edit the piece of content on the receiver side, you will have the same error on logs

Log: https://gist.github.com/bryanboza/5c618fa9e5c26065872a65fbee7030b3

**FYI: Tested doing a full reindex and that is not solving the problem** 

## Steps to Reproduce (for bugs)
1. Configure your pushing environment (sender//receiver )
2. Create two content types with a legacy relationship 
3. Add a related piece of content 
4. Push it 
5. Now upgrade the relationship on the sender 
6. Force push again the same piece of content 
7. Go to the receiver side and try to open the piece of content and check your log file 

## Your Environment
Tested on master // Postgres // FF... This on both sides

**Comment by wezell** [comment]
Does it go away when you delete the legacy rel on the receiver?

**Comment by bryanboza** [comment]
No, we still having the same error

**Comment by nollymar** [comment]
After several tests, I was able to see the error only once (before the reindex of the pushed contentlet finished).  However, I analyzed the exception trace and the error is thrown here https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java#L1399

This specific exception is already handled as a warning because it is just noise logged if you try to edit a contentlet while the reindex is updated (it does not affect future operations on that contentlet). This noise can be avoided in two ways: 

1. Removing the exception from the Logger.warn 
or
2. Querying the DB to verify if the relationship exists before hitting the index, which doesn't make any sense

After discussing these options with Rex Team, we consider them more expensive than having this temporal noise on the logs. So, no change will be applied.
